internal/securitate/userTabler.go

```diff
@@ -112,18 +112,18 @@ func (dataBase *DBstruct) UpdateToken(ctx context.Context, userName string, toke
 	return nil
 }
 
-func (dataBase *DBstruct) GetToken(ctx context.Context, userName string, tokenString *string) error {
+func (dataBase *DBstruct) GetToken(ctx context.Context, userName string) (string, error) {
 	db := dataBase.DB
 	//				получить токен из токен-таблицы  где код пользователя равен коду юзера из юзер-таблицы с именем UserName
 	order := "SELECT token from " + "tokens" + " WHERE userCode = (select usercode from accounts where login = $1) ;"
 	row := db.QueryRow(ctx, order, userName)
 	var str string
 	err := row.Scan(&str)
 	if err != nil {
-		return fmt.Errorf("GT %w", err)
+		return str, fmt.Errorf("GT %w", err)
```

Comment:
Если ошибка возвращается в таком виде, то к ней уже невозможно будет применить сравнение по err == sql.ErrNoRows, придется использовать `errors.Is`. КМК проще сделать
```suggestion
		return str, err
```
Тем более, что `GT` скорее будет проинтерпретирован, как `greater`. Но можно и оставить так, только помнить про метод вычленения ошибки.